assembl/static2/js/app/components/administration/exportSection.jsx

```diff
@@ -0,0 +1,161 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n, Translate } from 'react-redux-i18n';
+import { Link } from 'react-router';
+import { compose, graphql } from 'react-apollo';
+import { FormGroup, Radio, FormControl } from 'react-bootstrap';
+import withLoadingIndicator from '../../components/common/withLoadingIndicator';
+
+import { getDiscussionId } from '../../utils/globalFunctions';
+import SectionTitle from './sectionTitle';
+import DiscussionPreferenceLanguage from '../../graphql/DiscussionPreferenceLanguage.graphql';
+
+type Props = {
+  languages?: Array<Object>,
+  exportType: string,
+  voteSessionId?: string
+};
+
+type State = {
+  exportLocale: ?string,
+  translate: boolean
+};
+
+class DumbExportSection extends React.Component<Object, Props, State> {
+  props: Props;
+
+  state: State;
+
+  static defaultProps = {
+    languages: null,
+    voteSessionId: null
+  };
+
+  static ExportLanguageDropDown = ({ languages, onSelect, activeKey }) => {
+    const activeLanguage = languages.filter(language => language.locale === activeKey)[0];
+    return (
+      <FormControl
+        className="export-language-dropdown"
+        componentClass="select"
+        onChange={({ target: { value } }) => {
+          onSelect(value);
+        }}
+        value={activeLanguage.locale}
+      >
+        {languages.map(lang => (
+          <option key={`locale-${lang.locale}`} value={lang.locale}>
+            {lang.name}
+          </option>
+        ))}
+      </FormControl>
+    );
+  };
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      exportLocale: null,
+      translate: false
+    };
+  }
+
+  selectExportLocale = (locale: string) => {
+    this.setState({ exportLocale: locale });
+  };
+
+  toggleTranslation = (shouldTranslate: boolean) => {
+    this.setState({ translate: shouldTranslate });
+  };
+
+  render() {
+    const { languages, exportType, voteSessionId } = this.props;
+    const isSurveyExport = exportType === 'survey';
+    const { translate } = this.state;
+    const debateId = getDiscussionId();
+    if (!debateId) return null;
+    const exportLocale = this.state.exportLocale || (languages && languages[0].locale);
+    let exportLink;
+    if (isSurveyExport) {
+      exportLink = `/data/Discussion/${debateId}/phase1_csv_export${translate && exportLocale ? `?lang=${exportLocale}` : ''}`;
+    } else if (!isSurveyExport && voteSessionId) {
+      exportLink = `/data/Discussion/${debateId}/widgets/${voteSessionId}/vote_results_csv`;
+    }
+    return (
+      <div className="admin-box survey-admin-export-section">
+        <SectionTitle title={I18n.t('administration.survey.2')} annotation={I18n.t('administration.surveyExport.annotation')} />
+        <div className="admin-content">
+          {isSurveyExport && (
```

Comment:
ha ok. So we have just one section at the high level of the menu with cases depending on the data that we want to export